src/components/NoClubsFound.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Button, Card, Heading, Image, Link } from "@hackclub/design-system";
+
+const Base = Card.extend.attrs({
```

Comment:
like the `Header`, this doesn't need to be a styled component. define `Base` as `Card.withComponent(Container)`, then put these props (including `maxWidth`) inline in the main component